README.md

```diff
@@ -1,14 +1,10 @@
-# jiskefet
+# Bookkeeping
+This repository contains the software for ALICE Bookkeeping.
 
-This repo contains the software for the 'ALICE Electronic Logbook'. https://alice-logbook.cern.ch/
-
-## Versioning
-
-example: ```R.Arsenic.sp1.v0.0.1```
-
-Format:
-- ```R.``` (release)
-- ```Arsenic.``` (team name)
-- ```sp1.``` (sprint 1)
-- ```v0.0.1``` (version major, minor, patch)
+## Contributors
 
+### Team Arsenic
+|:---:|:---:|:---:|:---:|:---:|
+|:---:|:---:|:---:|:---:|:---:|
```

Comment:
one too many?

Reply:
Nah, looks nice (and the user avatars are not in the header row)